reco_utils/recommender/newsrec/newsrec_utils.py

```diff
@@ -329,3 +329,40 @@ def newsample(news, ratio):
         return news + [0] * (ratio - len(news))
     else:
         return random.sample(news, ratio)
+
+
+def get_mind_data_set(type):
```

Comment:
there is already code for downloading and extracting mind https://github.com/microsoft/recommenders/blob/master/reco_utils/dataset/mind.py could you please use that instead?

Reply:
see an example of use here: https://github.com/microsoft/recommenders/blob/master/examples/02_model_content_based_filtering/dkn_deep_dive.ipynb